README.md

```diff
@@ -81,6 +81,26 @@ For complete list of available configuration options see [:help nvim-tree.setup]
 
 Each option is documented in `:help nvim-tree.OPTION_NAME`. Nested options can be accessed by appending `.`, for example [:help nvim-tree.view.mappings](doc/nvim-tree-lua.txt)
 
+Using the icons requires a [patched font](https://www.nerdfonts.com/). If you
+don't have one installed, you're probably seeing undesirable "?" box glyphs.
+To disable the display of icons altogether, you can use the following minimal
+configuration:
```

Comment:
This is mostly covered in `:help nvim-tree-introduction` but clarity is nice.

We can just add "To disable the display of icons altogether set each of |renderer.icons.show| to false." in that section.

Reply:
Good idea, thanks for the suggestion.

Reply:
Agreed

Reply:
Sounds good. I will make the changes shortly.